app/dashboard/templates/dashboard/index-vue.html

```diff
@@ -452,6 +452,63 @@ <h1 class="font-title font-weight-semibold">Hackathon Coming Soon!</h1>
+                                  <div class="text-left">
+                                    <a :href="project.work_url" target="_blank" class="btn btn-sm btn-gc-blue font-smaller-2 font-weight-semibold">View Project</a>
+
+                                    <a :href="project.bounty.url" class="btn-sm btn-outline-gc-blue font-smaller-2 font-weight-semibold">View Bounty</a>
+                                    <b-dropdown varitant="primary" toggle-class="btn-sm btn btn-gc-blue" split-class="btn-sm btn btn-gc-blue">
+                                     <template v-slot:button-content>
+                                       <i class='fas fa-comment-dots'></i>
+                                     </template>
+                                     <b-dropdown-item-button @click.prevent="openChat(profile.handle);" v-for="profile in project.profiles" aria-describedby="dropdown-header-label">
+                                       @ [[ profile.handle ]]
+                                     </b-dropdown-item-button>
+                                    </b-dropdown>
+                                  </div>
+                                </div>
+
+                                <div class="my-3 mb-2 text-left">
+                                  <b class="font-weight-bold font-smaller-3">Team Members</b>
+                                  <div class="mt-1">
+                                      <a v-for="profile in project.profiles" :href=`/profile/${profile.handle}` class="">
+                                        <b-img-lazy :src="profile.avatar_url" :alt=`"@${profile.handle}"` :title="profile.handle" width="30" height="30" class="rounded-circle"></b-img-lazy>
+                                      </a>
+                                  </div>
+                                </div>
+
+                                <div v-if="project.looking_members || project.message" class="my-3 looking-team">
+                                  <h5 v-if="project.looking_members"  class="info-card-title uppercase">Looking for team members</h5>
+                                  <p v-if="project.message" class="info-card-desc">
+                                    [[ project.message ]]
+                                  </p>
+                                </div>
+
+                                <div class="font-smaller-2 mt-4">
+                                  <b class="font-weight-bold">Sponsored by</b>
+                                  <img class="" width="20" :src="project.bounty.avatar_url" :alt="project.bounty.org_name" />
+                                  <a :href=`/profile/${project.bounty.org_name}`>[[ project.bounty.org_name ]]</a>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                        <h1 v-if="userProjects.length > 0" class="font-title title ml-2 mt-4">All Projects</h1>
```

Comment:
^ seems like `userProjects.length > 0` is repeated!
Wrap it within a `<template>`  to make the code cleaner